src/services/healthcareProfessionalService.ts

```diff
@@ -315,6 +316,15 @@ function validateNames(
         })
     }
 
+   if (!names?.length) {
```

Comment:
You could probably combine this with the conditional on line 310 since they're checking the same field. For example, `if (!names || !names.length)`